### PR TITLE
Configuration options documented

### PR DESCRIPTION
**Pull request by alexeyvy**
https://oat-sa.atlassian.net/browse/TAO-8616

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,99 @@ extension-tao-delivery
 ======================
 
 Extension to manage delivery
+
+# Configuration options:
```
Comment (anshul119) [inline README.md]: There should only be 1 main heading in the readme. In this case its the name of the extension, we should move all the other heading one level down.

File: README.md [hunk 2]
```diff
@@ -2,3 +2,99 @@ extension-tao-delivery
...
+
+## AttemptService.conf.php
+
+### Configuration option "states_to_exclude"
```
Comment (anshul119) [inline README.md]: if anything is coming from the code please use backticks (`) to render it in an inline code block to give it better readability

File: README.md [hunk 3]
```diff
@@ -2,3 +2,99 @@ extension-tao-delivery
...
+
+*Description :* when retrieving attempts (executions), those attempts with specified states won't be retrieved
+
+*Possible states :* 
+* http://www.tao.lu/Ontologies/TAODelivery.rdf#DeliveryExecutionStatusActive : active state
```
Comment (anshul119) [inline README.md]: same goes for the urls, please use backticks to give it better readability.

File: README.md [hunk 4]
```diff
@@ -2,3 +2,99 @@ extension-tao-delivery
...
+
+### Configuration option "extension"
+
+*Description:* an extension name for composing a return URL
+
+*Possible values:* 
+* Any TAO extension name
+
+*Value examples:* 
+* taoDelivery
```
Comment (anshul119) [inline README.md]: values for the config options should also be in backticks for the same reason (it being part of the code).

File: README.md [hunk 5]
```diff
@@ -1,4 +1,148 @@
...
+* Any TAO roles
+
+*Value examples:* 
+* `[ 'http://www.tao.lu/Ontologies/TAO.rdf#DeliveryRole' ]`
+
+### DeliveryServer.conf.php
+No options
+
+### execution_service.conf.php
+No options
```
Comment (tikhanovichA) [inline README.md]: This config needed for configuring storage for delivery executions. For example, if we want to use key-value storage, then we should update this config with the next lines
`return new taoDelivery_models_classes_execution_KeyValueService(array(
    'persistence' => 'redis'
));`

File: README.md [hunk 6]
```diff
@@ -1,4 +1,148 @@
...
+```bash
+php tao\scripts\installExtension oat-sa/extension-tao-delivery
+```
+
+As a system administrator you also install it through the TAO Extension Manager:
+- Settings (the gears on the right hand side of the menu) -> Extension manager
+- Select _taoDelivery_ on the right hand side, check the box and hit _install_
+
+## REST API
+[](https://openapi.taotesting.com/viewer/?url=https://raw.githubusercontent.com/oat-sa/extension-tao-delivery/master/doc/rest.json)
```
Comment (tikhanovichA) [inline README.md]: This link is not visible

Comment (alexeyvy) [reply README.md]: It's currently unavailable, but it's on all readme templates: https://github.com/oat-sa/readme-templates/tree/master/extension-core/

**Pull request merged**